examples/spark/spark-report-deco.py

```diff
@@ -24,12 +24,12 @@
     cv = 5
     use_algos = [["lgb"]]
     dataset_name = "lama_test_dataset"
-    path, task_type, roles, dtype = get_dataset_attrs(dataset_name)
+    dataset = get_dataset(dataset_name)
 
-    train_data, test_data = prepare_test_and_train(spark, path, seed)
+    train_data, test_data = prepare_test_and_train(dataset, seed)
 
     with log_exec_timer("spark-lama training") as train_timer:
-        task = SparkTask(task_type)
+        task = SparkTask(dataset.task_type)
```

Comment:
the [same](https://github.com/sb-ai-lab/SLAMA/pull/7/files#r1223141108)

Reply:
These interfaces follows interfaces of LightAutoML, which is an "origin" of SLAMA.
Not sure if it is a good idea to change these intrefaces. Need a discussion on this matter. 
(though SLAMA change them if it is necessary for performance purposes or for convenience related to Spark specific)